src/Symfony/Component/HttpClient/NoPrivateNetworkHttpClient.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpClient\Exception\InvalidArgumentException;
+use Symfony\Component\HttpClient\Exception\TransportException;
+use Symfony\Component\HttpFoundation\IpUtils;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+use Symfony\Contracts\HttpClient\ResponseStreamInterface;
+
+/**
+ * Decorator that blocks requests to private networks by default.
+ *
+ * @author Hallison Boaventura <hallisonboaventura@gmail.com>
+ */
+final class NoPrivateNetworkHttpClient implements HttpClientInterface, LoggerAwareInterface
+{
+    use HttpClientTrait;
+
+    const PRIVATE_SUBNETS = [
```

Comment:
public or private?

Reply:
public LGTM

Reply:
Why public?

Reply:
so that ppl can reuse it?
private works for me also!

Reply:
Ahhh, I thought you were talking about the name of the constant. Is there still time to return the name of the constant to PRIVATE_SUBNETS? Thank you!

Reply:
`private const PRIVATE_SUBNETS` works